src/views/LabwhereReception.vue

```diff
@@ -170,7 +173,17 @@ const scanBarcodesToLabwhere = async () => {
       start_position.value,
     )
     if (response.success) {
-      showAlert(response.message, 'success')
+      let message = response.message
+      // Check if the library volume need to be exhausted
+      const { success, exhaustedLibraries } = await exhaustLibraryVolumeIfDestroyed(
+        location_barcode.value,
+        uniqueBarcodesArray.value,
+      )
+      if (success && exhaustedLibraries.length > 0) {
```

Comment:
This message formatting should probably be handled by `exhaustLibraryVolumeIfDestroyed` as exhaustedLibraries are only used for the message.